firestore-perspective-toxicity/README.md

```diff
@@ -0,0 +1,13 @@
+# Analyze Toxicity with Perspective API
+
+**Author**: Jigsaw's Perspective API (**[https://perspectiveapi.com](https://perspectiveapi.com/)**)
+
+**Description**: We’ve partnered with the Jigsaw team to build the Analyze Toxicity extension, which leverages machine learning to classify the level of toxicity, threat and profanity of your user comments. The Perspective API uses the power of machine learning to help mitigate toxicity and ensure healthy dialogue online. It is trusted by sites like the New York Times or the Wall Street Journal. Please check it out [here](https://github.com/conversationai/firestore-perspective-toxicity).
```

Comment:
2nd sentence:
- Nit: remove "The" before 'Perspective API' 
- Add the link to the Perspective site (https://perspectiveapi.com/) after 'Perspective API' 
3rd sentence:
- Change 'sites' to 'platforms'
- Please change 'New York Times or the Wall Street Journal' to 'New York Times and Reddit'